### DISCUSSION TITLE
Cara membuat halaman tertentu menjadi Full AMP tanpa direct ke '/amp/'?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#241] [Q&A] [answered]
Bagaimana cara mengubah halaman menjadi full amp tanpa mengganggu halaman lain yang semi AMP?

misalnya saya ingin menjadikan halaman 'domain.com/stories' ini menjadi halaman full amp tanpa harus di direct ke halaman yang menggunakan '/amp/*' dengan menggunakan penulisan kode sesuai yang ada di diskusi #137

**Comment by yuliakbard** [comment]
berikut halaman amp yang ingin saya ubah jadi full AMP
`https://www.lokersumatera.com/amp/stories/1`

**Comment by taufik-nurrohman** [comment, marked as answer]
~~~ php
if ('/stories/1' === $url->path) {
    State::set('is.mobile', true);
}
~~~
